src/main/java/SoloTech/StockFlow/common/cache/CacheSubscriber.java

```diff
@@ -0,0 +1,37 @@
+package SoloTech.StockFlow.common.cache;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.redis.connection.Message;
+import org.springframework.data.redis.connection.MessageListener;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.data.redis.serializer.StringRedisSerializer;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class CacheSubscriber implements MessageListener {
+
+    private final Cache<String, Object> localCache;
+    private final RedisTemplate<String, Object> redisTemplate;
+
+    @Override
+    public void onMessage(Message message, byte[] pattern) {
+        StringRedisSerializer serializer = new StringRedisSerializer();
+        String body = serializer.deserialize(message.getBody());
+
+        log.info("Received message: {}", body);
+
+        assert body != null;
```

Comment:
```suggestion
        if (body == null)
```

assert 는 런타임에서 예외가 안잡힐텐데요

assert 를 사용하신 이유가 있을까요?